BlazorMarkDownAppJwt/BlazorMarkDownAppJwt/Server/Services/MarkDowns/DocumentService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using BlazorMarkDownAppJwt.Server.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace BlazorMarkDownAppJwt.Server.Services.MarkDowns
+{
+    public class DocumentService : IDocumentService
+    {
+
+        private readonly DataBaseContext ctx;
+
+        private readonly IWebHostEnvironment env;
+
+        private const string documentPath = "Datas";
+
+
+        public DocumentService(DataBaseContext ctx, IWebHostEnvironment env)
+        {
+            this.ctx = ctx;
+            this.env = env;
+        }
+        public async Task<Document?> GetDocumentAsync(long id, CancellationToken cancellationToken)
+        {
+            var document = await ctx.Document.FirstOrDefaultAsync(p => p.Id == id, cancellationToken);
+            return document;
+        }
+
+        public async Task<Document> InsertDocumentAsync(string markDown, CancellationToken cancellationToken)
+        {
+            Document document = new Document
+            {
+                MarkDown = markDown
+            };
+            ctx.Add(document);
+
+            await ctx.SaveChangesAsync(cancellationToken);
+
+            return document;
+        }
+
+        public async Task<Document?> GetReadMeDocumentAsync(CancellationToken cancellationToken)
+        {
+            var path = Path.Combine(env.ContentRootPath, documentPath);
+            if (!Directory.Exists(path))
+                return null;
+            path = Path.Combine(path, "README.md");

```

Comment:
why it's not comes from db? you made and editor tool, but made a doc in dedicated file
dubious product advertising :D